ios/MagnetScannerReact.swift

```diff
@@ -14,12 +17,41 @@ import Foundation
   }
   
   @objc func stop() {
-    scanner.stop()
+    // We don't stop scanning in iOS, we let the OS to
+    // throttle the notifications of beacons found.
+    // scanner.stop()
   }
   
   func onItemFound(item: Dictionary<String, AnyObject>) {
-    print("item found", item["url"]);
+    print("item found", item["url"])
     bridge.eventDispatcher()
-      .sendDeviceEventWithName("magnetscanner:itemfound", body: item);
+      .sendDeviceEventWithName("magnetscanner:itemfound", body: item)
+    
+    notifyUser(item["url"] as! String)
+  }
+  
+  // Throttles the notification process, waiting for 1 minute until
+  // setting up the badge with the number of elements nearby.
+  func notifyUser(url: String) {
+    if (toNotify.contains(url)) {
+      return;
+    }
```

Comment:
As you want to dedupe on url and order doesn't matter, would a `Dictionary` make more sense instead of an `Array`?